src/main/java/uk/gov/ons/census/action/model/entity/ActionType.java

```diff
@@ -91,6 +91,8 @@
 
   P_UAC_IX(ActionHandler.PRINTER), // Individual response UAC print
 
+  P_UAC_CX(ActionHandler.PRINTER), // CE UAC via paper
```

Comment:
This might seem petty, but I don't love it when this class starts to differ from the one in scheduler and worker - `P_ER_IL` was the last one on the list in the other two